sdk/communication/AzureCommunication/Source/CommunicationIdentifierModel.swift

```diff
@@ -0,0 +1,82 @@
+// --------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// --------------------------------------------------------------------------
+
+import AzureCore
+import Foundation
+// swiftlint:disable superfluous_disable_command
+// swiftlint:disable identifier_name
+// swiftlint:disable line_length
+// swiftlint:disable cyclomatic_complexity
+
+/// Identifies a participant in Azure Communication services. A participant is, for example, a phone number or an Azure communication user.
+public struct CommunicationIdentifierModel: Codable {
+    // MARK: Properties
+
+    /// Kind of the communication identifier.
+    public let kind: CommunicationIdentifierKind
+    /// Full Id of the identifier.
+    public let id: String?
```

Comment:
Although not a super big deal `id` is a reserved type in objective-c. I think we should change this to `identifier`. 
We do the same thing in the identifiers class.

Reply:
Enumerations.swift and CommunicationIdentifierModel.swift are generated code from AutoRest swift.